src/skmultiflow/trees/hoeffding_adaptive_tree.py

```diff
@@ -34,7 +34,6 @@ def HAT(max_byte_size=33554432, memory_estimate_period=1000000, grace_period=200
 MAJORITY_CLASS = 'mc'
 NAIVE_BAYES = 'nb'
 NAIVE_BAYES_ADAPTIVE = 'nba'
```

Comment:
Please make these constants private, so they are not shown in the documentation.

Reply:
Addressed in commit [f5838ee](https://github.com/scikit-multiflow/scikit-multiflow/pull/220/commits/f5838ee9b96b27c674a13a529e065d3b91c38aa2).